serialization/python/json/serialization/json_parse_node_factory.py

```diff
@@ -0,0 +1,39 @@
+import json
+from io import BytesIO
+
+from kiota.abstractions import ParseNode, ParseNodeFactory
+
+from .json_parse_node import JsonParseNode
+
+
+class JsonParseNodeFactory(ParseNodeFactory):
+    """Factory that is used to create JsonParseNodes.
+    """
+    def get_valid_content_type(self) -> str:
+        """Returns the content type this factory's parse nodes can deserialize
+        Returns:
+            str: The content type to be deserialized
+        """
+        return 'application/json'
+
+    def get_root_parse_node(self, content_type: str,
+                            content: BytesIO) -> ParseNode:
+        """Creates a ParseNode from the given binary stream and content type
+        Args:
+            content_type (str): The content type of the binary stream
+            content (BytesIO): The array buffer to read from
+        Returns:
+            ParseNode: A ParseNode that can deserialize the given binary stream
+        """
+        if not content_type:
+            raise TypeError("Content Type cannot be null")
+        if self.get_valid_content_type() != content_type:
```

Comment:
casing

Reply:
Updated in https://github.com/microsoft/kiota/pull/1186/commits/8f030114b3625116336c7abfe57d57c8d1fdc486